modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -167,3 +183,67 @@ START_TEST(PrivateExtractIcons)
         DeleteFileW(PathBuffer);
     }
 }
+
+static const struct tagPAIRSTESTS
+{
+    BYTE InCount;
+    BYTE UseHigh; // Extract a pair (high and low word sizes)
+    BYTE Library;
```

Comment:
```suggestion
    BOOL UseHigh; // Extract a pair (high and low word sizes)
    BOOL Library;
```
and related changes.

Reply:
Then someone will come along and demand that I use TRUE and FALSE and cause misalignment in the table :)